jbig2/jbig2_compress_image.go

```diff
@@ -1,12 +1,6 @@
 /*
  * This example showcases the conversion of the jpg encoded image
  * into jbig2 encoding format.
- *
- * The result jbig2 file is compressed with a lossless method stored with
```

Comment:
Maybe keep this but say. For example, with the provided file xxx.jpg ....

Reply:
No need to keep this now. Done.